docs/developer_interface.rst

```diff
@@ -16,12 +16,6 @@ Page
 .. automodule:: pomcorn.page
     :members:
 
-Element
```

Comment:
Let's restore docs for `PomcornElement` and add note that this class is returned when using descriptors